### PR TITLE
[4.0] Web services

### PR DESCRIPTION
**Pull request by brianteeman**
Ref: #31306 This pr standardises the term to be "Web services". Note the capitalisation. This is inline with the naming used by the W3C.

The style guide rules of capitalising the words in a label still apply so other strings do not need to be updated.

I will submit an additional PR to style guide repo

### CODE REVIEW

File: administrator/language/en-GB/com_config.ini [hunk 1]
```diff
@@ -179,8 +179,8 @@ COM_CONFIG_FIELD_VALUE_SYSTEM_DEFAULT="System Default"
 COM_CONFIG_FIELD_VALUE_TLS="STARTTLS"
 COM_CONFIG_FIELD_WEBSERVICES_CORSOFFF_LABEL="Enable CORS"
 COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_HEADERS_DESC="Specifies the header(s) sent back in response to a preflight request. Default: Content-Type,X-Joomla-Token"
-COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_METHODS_DESC="Specifies the webservice method(s) allowed to access on this site, sent back in response to a preflight request. Default: all methods available for the requested route."
-COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_ORIGIN_DESC="Specifies the origin allowed to access webservices on this site,sent back in response to a preflight request. Default: * (=all)."
+COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_METHODS_DESC="Specifies the Web service method(s) allowed to access on this site, sent back in response to a preflight request. Default: all methods available for the requested route."
+COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_ORIGIN_DESC="Specifies the origin allowed to access Web services on this site,sent back in response to a preflight request. Default: * (=all)."
```
Comment (Quy) [inline administrator/language/en-GB/com_config.ini]:
```suggestion
COM_CONFIG_FIELD_WEBSERVICES_CORS_ALLOW_ORIGIN_DESC="Specifies the origin allowed to access Web services on this site, sent back in response to a preflight request. Default: * (=all)."
```

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on 62eb7d1aec4731440025ee841938a728bcf006ac<br /><br />Looked at diff and applied PR - saw Config / Server label change to Web Services. Noticed that the Logging tab has a 'Custom logging' panel (small l). Are there different rules for Legends and Labels? Enabled CORS to see some of the labels with 'Web service'.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32496">issues.joomla.org/tracker/joomla-cms/32496</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 62eb7d1aec4731440025ee841938a728bcf006ac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32496">issues.joomla.org/tracker/joomla-cms/32496</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32496">issues.joomla.org/tracker/joomla-cms/32496</a>.</sub>

**Comment by drmenzelit** [comment]
Thanks

**Comment by brianteeman** [comment]
thanks

**Pull request merged**